src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13107,15 +13100,12 @@ private static Vector256<T> CreateAlignmentMaskVector256<T>(int count)
                     (uint)(count * 16));
             }
 
-            if (Unsafe.SizeOf<T>() == 8)
+            Debug.Assert(Unsafe.SizeOf<T>() == 8);
             {
```

Comment:
Nit: shouldn't the braces be removed as well?

Reply:
I kept them to keep the structure the same as the other blocks. Not a strong preference, though.

Reply:
(This particular block doesn't define any locals, but it's consistent with others that do, e.g. https://github.com/dotnet/runtime/pull/98281/files/e0c3aca1a2d096ec001af024ef5027fa44723d98#diff-9af9871a07c3bc8868a023467dd1e92b08eaee3e04fcc490126d89263678ed8eR18788-R18800, where they need to be in a scope to avoid conflicting with other locals in the method.)